src/routes/register/registerData.routes.ts

```diff
@@ -0,0 +1,71 @@
+import {RegisterDTO} from './registerDTO'
+import {bodyValidator} from '../../shared/bodyValidator'
+import bcrypt from 'bcrypt'
+import jwt from 'jsonwebtoken'
+import {config} from 'dotenv'
+import Register from './registerModel'
+import {Router} from 'express'
+import {User} from '../../types/customUser'
+
+config()
+
+const registerRouter = Router()
+
+registerRouter.post('/api/register', async(req, res) => {
+	try {
+		await bodyValidator(RegisterDTO, req.body)
+		const birth = req.body.birthDate
+		const date18YearsAgo = new Date()
+		date18YearsAgo.setFullYear(date18YearsAgo.getFullYear()-18)
+		if(new Date(birth) >= date18YearsAgo) {
+			return res.status(401).json({message: 'You have to be at least 18 years old'})
+		}
+		const userLogin = await Register.findOne({login: req.body.login}, 'login').exec()

```

Comment:
Jak zmienisz nazwę Scheme to zmień też jej użycia z Register na nazwę pochodzącą od nazwy modelu

Reply:
zmieniono na UserAccount